Source/DafnyServer/SuperLegacySymbolTable.cs

```diff
@@ -227,9 +227,7 @@ private static void ParseUpdateStatement(Statement statement, List<SymbolInforma
       var updateStmt = (AssignStatement)statement;
       var leftSide = updateStmt.Lhss;
       var rightSide = updateStmt.Rhss;
-      var leftSideDots = leftSide.OfType<ExprDotName>();
-      var rightSideDots = rightSide.OfType<ExprDotName>();
-      var allExprDotNames = leftSideDots.Concat(rightSideDots);
+      var allExprDotNames = leftSide.OfType<ExprDotName>();
```

Comment:
Why did this change?

Reply:
@atomb made this change. Rider confirms it's a noop:

<img width="905" alt="image" src="https://github.com/user-attachments/assets/69911a0a-861b-49c2-ada2-36b8e8c0a721" />

Same for the other one I imagine.

Reply:
Ah got it, definitely not obvious without context